package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
What is this?

Reply:
I'm not sure why that file is there.. maybe it was created during installing packages or something. I will remove the file.